README.md

```diff
@@ -225,6 +227,80 @@ When using backreferences, there is a couple of things to keep in mind:
 - `__via__` is surrounded with two underscores; you can read this query out loud like this: _"get posts that link to author via the author field"_
 
 
+## Helpers
+
+`cf-graphql` comes with helpers that help you with the `cf-graphql` integration. These are used inside of [the demo application](https://github.com/contentful-labs/cf-graphql/tree/master/demo).
+
+
+### `expressGraphqlExtension`
+
+If you're using [express-graphql](https://github.com/graphql/express-graphql) `cf-graphql` provides you an additional extension that you could use to enable additional information.
+
+```javascript
+const opts = {
+  // retreive the current cf-graphql version
+  version: true,
+  // include timings of the underlying Contentful CDA calls
+  timeline: true,
+  // format errors in a nice way
```

Comment:
Mention `timeline` and `detailedErrors` should be set to `false` when in production